zerver/lib/onboarding.py

```diff
@@ -4,11 +4,24 @@
 from zerver.lib.actions import set_default_streams, bulk_add_subscriptions, \
     internal_prep_stream_message, internal_send_private_message, \
     create_stream_if_needed, create_streams_if_needed, do_send_messages, \
-    do_add_reaction_legacy
+    do_add_reaction_legacy, create_users
 from zerver.models import Realm, UserProfile, Message, Reaction, get_system_bot
 
 from typing import Any, Dict, List, Mapping, Text
 
+def setup_realm_internal_bots(realm: Realm) -> None:
+    internal_bots = [(bot['name'], bot['email_template'] % (settings.INTERNAL_BOT_DOMAIN,))
```

Comment:
Is there a reason we can't use bot['email'] here?  We seem to calculate it as part as `settings.py`.